### PR TITLE
LPS-126254

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Fieldsets.testcase [hunk 1]
```diff
@@ -123,12 +123,51 @@ definition {
 	}
 
 	@description = "This is a test for LPS-107630. As an App Adm, I want to use an existing fieldset"
-	@ignore = "true"
 	@priority = "4"
 	test RepeatFieldsetWithoutDuplicatingContent {
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Fieldsets.testcase:127]: The test points is repeat fieldsets while the case is repeat field

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Fieldsets.testcase [hunk 2]
```diff
@@ -123,12 +123,51 @@ definition {
...
 
-		// TODO LPS-107630 RepeatFieldsetWithoutDuplicatingContent pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			panel = "Site Administration",
+			portlet = "Web Content");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Fieldsets.testcase:137]: For example, add a web content structure with a Basic Web Content fieldset

**Comment by alexcandidos** [comment]
hey @emanuelfisilva , please take a look when possible! Thanks!

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3615
View total diff https://github.com/liferay-echo/liferay-portal/pull/3615/commits/883db76b1c269ad21734adaecfc8605058f1c3d2

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**